README.md

```diff
@@ -1,30 +1,26 @@
-# ECMAScript proposal: JavaScript standard library UUID module
+# ECMAScript proposal: JavaScript standard library UUID built-in
```

Comment:
Nit: I would delete this trailing word "built-in", just because it's not a common phrasing. You can instead just say "standard UUID library" or (below) "a candidate for the JavaScript standard library".

Reply:
Maybe the intention was to already pick up the term `builtin module` as proposed in the 4th-last slide of https://github.com/tc39/proposal-javascript-standard-library/blob/master/slides/JSL-TC39-June-2019.pdf ?

If that is the intention, then it should indeed read `JavaScript builtin UUID module`.

As far as I understand the discussion in https://github.com/tc39/proposal-javascript-standard-library/issues/50 the current proposal is to implement the standard library using modules and not (global scope) builtins. I suggest to follow the current status of the standard library proposal and not take side for those who propose to implement the standard library as `builtins` instead of `modules`.

So if we want to anticipate the proposed `standard library modules` -> `builtin modules` renaming, I suggest we write `JavaScript builtin UUID module` but keep the assumption that we'll get `modules` in the end and not `builtins`.

Reply:
> I suggest to follow the current status of the standard library proposal and not take side for those who propose to implement the standard library as builtins instead of modules.

I agree with the idea to not take sides on the built-in module debate here. I am just narrowly commenting on the wording--we have been adding a number of functions to JS over the years, and we tend to not use the word "built-in" as a noun this way. So I'd avoid this wording to minimize potential confusion.